tests/foreman/api/test_syncplan.py

```diff
@@ -953,6 +953,7 @@ def test_positive_synchronize_rh_product_future_sync_date(self):
         self.validate_repo_content(
             repo, ['erratum', 'package', 'package_group'])
 
+    @upgrade
```

Comment:
I think this functionality is already covered by CLI
https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/cli/test_syncplan.py#L767

Reply:
:+1: